Our Mumble build environments have zlib
headers in -I/MumbleBuild/${name}/include

Qt doesn't pick them up when building
qtbase/src/tools/bootstrap.

This somewhat hacky patch solves the
problem on our CentOS buildenv.

--- ./qtbase/src/tools/bootstrap/bootstrap.pro
+++ ./qtbase/src/tools/bootstrap/bootstrap.pro
@@ -143,6 +143,7 @@
 }
 
 load(qt_module)
+load(qt_build_extra)
 
 # otherwise mingw headers do not declare common functions like putenv
 mingw: CONFIG -= strict_c++
